libsrc/apple2/get_ostype.s

```diff
@@ -67,4 +83,13 @@ _get_ostype:
 
         .segment        "INIT"
 
-ostype: .res    1
+ostype:         .res    1
+iie_or_newer:   .res    1
```

Comment:
I think this - with a different name - want's to be inside an `.ifndef __APPLE2ENH__`.